charts/nextcloud/templates/metrics-deployment.yaml

```diff
@@ -51,7 +51,7 @@ spec:
               key: {{ .Values.nextcloud.existingSecret.passwordKey | default "nextcloud-password" }}
         {{- end }}
         - name: NEXTCLOUD_SERVER
-          value: http{{ if .Values.metrics.https }}s{{ end }}://{{ .Values.nextcloud.host }}
+          value: http{{ if .Values.metrics.https }}s{{ end }}://{{ template "nextcloud.fullname" . }}:{{ .Values.service.port }}
```

Comment:
Can you add a comment above this line that documents that this points to the service in https://github.com/nextcloud/helm/blob/b35eaa38a721542d7ce73d07e123aa0574244c26/charts/nextcloud/templates/service.yaml#L4 ?

Reply:
I meant more about the resolution of the address of the service (explaining that `template "nextcloud.fullname" .` is used to resolve the service host)

Reply:
Isn't that what k8s-services are generally there for?

Reply:
Yes, but I wanted a comment like `"nextcloud.fullname" resolves the hostname of the nextcloud service`.

Reply:
I provided a fix for the bug, its up to you what you do with it. I don't see such a comment useful since its obvious where the service gets its name from, what a K8s-Service is for and therefore I wont write it.